tests/test_data/packages/flit_small_with_deps/module/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""Sample module
+"""
+
+__version__ = "0.1.1"  # pragma: no cover
```

Comment:
We try to avoid creating fixture packages in `test/test_data` and encourage to create packages dynamically in the test body or pytest fixtures instead (see [make_package](https://github.com/jazzband/pip-tools/blob/77aa19b63c9e2aad410e6f5654c19f78b8101ddf/tests/conftest.py#L242)). 

Ideally, it would be nice to hive a fixture like `make_flit_package(name, version, install_requires, extras, ...)` to avoid boilerplate.

Reply:
But if you have no time or energy to spend on that, we could leave it as is and refactor later. I don't mind 👍🏻

Reply:
It's part of metadata, flit requires it. Or do you mean, generate any metadata (including `pyproject.toml`) as well? It's hard to tell which way is preferred now, some packages use `make_package`, some use `PACKAGES_PATH`. I'm not sure what would be better here. Having pyproject.toml as a separate TOML file gives a small benefit of syntax highlighting, formatting all that nice stuff 🤔

Reply:
Fixed.